src/compiler/proposals-and-stabilization.md

```diff
@@ -0,0 +1,336 @@
+- Removing flags which were intended for eventual stabilization
+  - **Propose using:** MCP
+  - **Approve using:** Seconding
+  - Describe the rationale for removing the unstable implementation. Once discussion has concluded,
+    a team member may second the proposal
+- Stabilizing a compiler option
+  - **Propose using:** PR
+  - **Approve using:** FCP
+  - Open a PR and follow the [stabilization guide][stabilization_guide]. The assigned reviewer will
+    check that the stabilization guide has been followed, review the code and start an FCP
+- Reverting stabilization of a compiler option
+  - **Propose using:** PR
+  - **Approve using:** FCP
+  - Open a PR and follow the [stabilization guide][stabilization_guide]. The assigned reviewer will
+    check that the stabilization guide has been followed, review the code and start an FCP
+
+### Attributes
+
+- Adding a attribute for internal-use only (e.g. `rustc_attrs`)
+  - **Propose using:** PR
+  - **Approve using:** r+
+  - If a team member finds the new attribute reasonable then they can merge the change adding the
+    attribute
+- Adding a attribute with intent to later stabilize
+  - Follow the language team's process and have the implementation PR reviewed by a member of the
+    compiler team
+- Removing internal-use only attributes
+  - **Propose using:** MCP
+  - **Approve using:** Seconding
+  - Describe the rationale for removing the unstable implementation. Once discussion has concluded,
+    a team member may second the proposal
+- Removing attribute which were intended for eventual stabilization
+  - Follow the language team's process and have the removal PR reviewed by a member of the compiler
+    team
+- Stabilizing an attribute
+  - Follow the language team's process and have the stabiization PR reviewed by a member of the
+    compiler team
+- Reverting stabilization of an attribute
+  - Follow the language team's process and have the revert PR reviewed by a member of the
+    compiler team
+
+### Features
+
+- Adding experimental implementations of not-yet-proposed language features
+  - **Propose using:** MCP
+  - **Approve using:** Seconding
+  - With the approval of the language team (that they think the feature is worth experimentation),
+    then submit an RFC and if, after discussion has concluded, a compiler team member agrees that
+    the implementation is feasible and will not put undue burden on the maintainers of the compiler,
+    then they can second the MCP and implementation can proceed.
+  - This isn't necessary if the owner of the implementation is a member of the compiler team
+- Stabilizing a language feature
+  - Follow the language team's process and have the stabiization PR reviewed by a member of the
+    compiler team
+- Reverting stabilization of a language feature
+  - Follow the language team's process and have the revert PR reviewed by a member of the
+    compiler team
+
+### Targets
+
+- Proposing a new target
```

Comment:
Suggestion: if a new target diverges significantly with existing targets (prime example is the amdgpu targets that have different `addrspace`s), then MCP + seconding for the proposal, then r+ from compiler leads if MCP is accepted?

Reply:
Fixed!